src/supervised.jl

```diff
@@ -1467,3 +1467,14 @@ true
 issymmetric(::SupervisedLoss) = false
 
 isminimizable(l::SupervisedLoss) = isconvex(l)
+
+# -----------------
+# IMPLEMENTATIONS
```

Comment:
I would include these from LossFunctions.jl.  Nested `include`s  makes it harder to find things.

Reply:
I moved them here because then people know that these files are a "subset" of the "supervised.jl" concept. If we think that we will add more types of losses in the future, this approach scales well, and we don't have hundreds of includes in `LossFunctions.jl`. Would it be ok to leave the includes here? I can move them to the top level as well if you prefer.

Reply:
It's already a huge file (>1400 lines).  Let's put them in LossFunctions.jl so they're easier to find.

Reply:
I think the issue is that currently `supervised.jl` is too long. I will refactor the code further to split it into smaller digestible files. Hopefully the includes will become more visible.

Reply:
Ok, now the supervised.jl file only contains a few provisory implementations of the traits in LearnBase.jl plus the includes for specific losses. This file will likely disappear in the near future after a planned refactoring in LearnBase.jl.